scripts/docker-compose.yml

```diff
@@ -1,22 +1,22 @@
-version: '3.7'
+version: "3.7"
 
 services:
   postgresql:
     container_name: postgres
     image: library/postgres:12
```

Comment:
```suggestion
    image: docker.io/library/postgres:12
```

Reply:
We could update this to 16, but it would probably annoy a bunch of us. Let's perhaps wait until just after a release.